homeworks/olena.ambitska_elena-ambitska/4-basics/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Basic JS</title>
+    </head>
+    <body>
+        <h1>Basic math operation</h1>
```

Comment:
Good (and so speedy) work!
Please add styles, as described in the [task](https://github.com/MastersAcademy/fe-course-basic-2023/issues/184#issue-1963547988).
>  ...із стандартними стилямі заголовка з вашего проєкту.

Reply:
@vladyslav-yermolin-moc thanks for the review. Added styles and a little bit of interactives with the user :) 

<details>

<img width="1080" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/74259586/ecaaeaa9-3ae9-40c2-8a1b-06222a3c99bf">


</details>